deploy/lvm-operator.yaml

```diff
@@ -446,6 +446,8 @@ spec:
   # do not require volumeattachment
   attachRequired: false
   podInfoOnMount: true
+  storageCapacity: true
```

Comment:
this could fail to apply in k8s 1.18 version since the [storageCapacity get added in k8s 1.19 (alpha)](https://kubernetes.io/docs/concepts/storage/storage-capacity/).

Workaround is to delete the `CSIDriver` resource and apply the yaml again to upgrade the control plane components

Reply:
Just seen that we have minimum k8s version 1.20+ in lvm localpv ,